README.md

```diff
@@ -1,23 +1,77 @@
+
+
+# Running
+Operating an EVM Gateway is straightforward. It can either be deployed locally alongside the Flow emulator or configured to connect with any active Flow networks supporting EVM. Given that the EVM Gateway depends solely on [Access Node APIs](https://developers.flow.com/networks/node-ops/access-onchain-data/access-nodes/accessing-data/access-api), it is compatible with any networks offering this API access.
+
+### Running Locally
+**Start Emulator**
+
+In order to run the gateway locally you need to start the emulator with EVM enabled:
+```
+flow emulator --evm-enabled
+```
+_Make sure flow.json has the emulator account configured to address and private key we will use for starting gateway bellow._
+
+Then you need to start the gateway:
+```
+go run cmd/main/main.go 
+  --init-cadence-height 0 
+  --coinbase FACF71692421039876a5BB4F10EF7A439D8ef61E 
+  --coa-address f8d6e0586b0a20c7 
+  --coa-key 2619878f0e2ff438d17835c2a4561cb87b4d24d72d12ec34569acd0dd4af7c21 
+  --coa-resource-create 
+  --gas-price 0
+```
+
+_In this example we use `coa-address` value set to service account of the emulator, same as `coa-key`. 
+This account will by default be funded with Flow which is a requirement. For `coinbase` we can 
+use whichever valid EVM address. It's not really useful for local running beside collecting fees. We provide also the 
+`coa-resource-create` to auto-create resources needed on start-up on the `coa` account in order to operate gateway. 
+`gas-price` is set at 0 so we don't have to fund EOA accounts. We can set it higher but keep in mind you will then 
+need funded accounts for interacting with EVM._
+
+## Configuration Flags
+
+The application can be configured using the following flags at runtime:
+
+| Flag                       | Default Value    | Description                                                                                                            |
+|----------------------------|------------------|------------------------------------------------------------------------------------------------------------------------|
+| `--database-dir`           | `./db`           | Path to the directory for the database.                                                                                |
+| `--rpc-host`               | (empty)          | Host for the JSON RPC API server.                                                                                      |
+| `--rpc-port`               | `3000`           | Port for the JSON RPC API server.                                                                                      |
```

Comment:
I think we should keep `8545` as the default value, as this is the traditional port that EVM devs are familiar with.